contracts/minters/vending-minter-wl-flex/src/contract.rs

```diff
@@ -730,9 +730,15 @@ fn _execute_mint(
         .add_attribute("sender", info.sender)
         .add_attribute("recipient", recipient_addr)
         .add_attribute("token_id", mintable_token_mapping.token_id.to_string())
-        .add_attribute("network_fee", network_fee)
-        .add_attribute("mint_price", mint_price.amount)
-        .add_attribute("seller_amount", seller_amount))
+        .add_attribute(
+            "network_fee",
+            coin(network_fee.u128(), mint_price.clone().denom).to_string(),
```

Comment:
I don't think we need to change network fee if it is a coin already the to_string() implementation should be including amount and denom

Reply:
`network_fee` is only a Uint128 here I am afraid.